docs/Model_Test.md

```diff
@@ -9,8 +9,8 @@ You should have:
 You can get onnx python package from [pypi](https://pypi.org/). However, if you are a onnxruntime developer, you may need to work on a cutting edge ONNX version. In this case, you need to build and install ONNX from source code.
 
 ### Install ONNX from source code
```

Comment:
Instead of duplicating some information that may get out of date, would it be better to just link to the relevant sections in the ONNX documentation?

Reply:
They don't have a good user doc yet.